google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -100,6 +102,46 @@ public void trackEvent(@NotNull String eventCategory,
     }
   }
 
+  @Override
+  public PartialTrackingEventAction trackEvent(String category) {
+    return new TrackingEventBuilder(category);
+  }
+
+  private class TrackingEventBuilder implements
+      PartialTrackingEventAction, PartialTrackingEventLabel, PartialTrackingEventValue {
+    private String category;
+    private String action;
+    private String label;
+    private Integer value;
+
+    public TrackingEventBuilder(String category) {
```

Comment:
you're right I think that it should be private.

On Wed, Jun 22, 2016 at 2:01 PM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/752#discussion_r68103396>
> :
>
> > @@ -100,6 +102,46 @@ public void trackEvent(@NotNull String eventCategory,
> >      }
> >    }
> >
> > +  @Override
> > +  public PartialTrackingEventAction trackEvent(String category) {
> > +    return new TrackingEventBuilder(category);
> > +  }
> > +
> > +  private class TrackingEventBuilder implements
> > +      PartialTrackingEventAction, PartialTrackingEventLabel, PartialTrackingEventValue {
> > +    private String category;
> > +    private String action;
> > +    private String label;
> > +    private Integer value;
> > +
> > +    public TrackingEventBuilder(String category) {
>
> Never mind, you aren't using it completely like a traditional builder.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/752/files/baccf888600f472b69946034e2bc213cc9ce67ec#r68103396>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HW-_luLNfQWflYOuxdDsDaUhHK4Nks5qOXiNgaJpZM4I8B5a>
> .
>